src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -79,6 +79,18 @@ public sealed class Arguments : TransformInputBase
 
         private readonly SchemaImpl _schemaImpl;
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
```

Comment:
>Input <see cref="IDataView"/>. This is the output from previous transform or loader [](start = 32, length = 83)

This on the other hand might be over-description. Certainly this is not the best place for people to learn how to instantiate `IDataView`s if they don't already know how to do so. in It's also not *quite* correct, since there are certainly other data views other than loaders and transforms.

Reply:
ok, I will update it as a part of this issue #524.

---
In reply to: [201911091](https://github.com/dotnet/machinelearning/pull/520#discussion_r201911091) [](ancestors = 201911091)